app.js

```diff
@@ -4,46 +4,58 @@ import { getRandomPokemon } from './utils.js';
 
 const radioButtons = document.querySelectorAll('input');
 const images = document.querySelectorAll('label > img');
-const travel = document.querySelector('#move');
 const capture = document.querySelector('#capture-button');
 
 
 
-// initialize state
+// initialize states
 const pokemon = pokemonData.slice();
 let capturedPoke = 0;
-let encounteredPoke = 0;
-const missedPokemon = [];
+let encounteredPoke = 0; 
 
-// set event listeners to update state and DOM
+let encounteredPokeArray = [];
+let missedPokemonArray = [];
 
-// let ashesPicked;
+// set event listeners to upddate state and DOM
 
-// radioButtons.forEach();
 
-let pokeOne = getRandomPokemon(pokemonData);
-let pokeTwo = getRandomPokemon(pokemonData);
-let pokeThree = getRandomPokemon(pokemonData);
+capture.addEventListener('click', () => {
+    capturedPoke++;
+    encounteredPoke++;
+    encounteredPoke++;
+    encounteredPoke++;
+    refreshPoke();
+});
 
-while (pokeOne.id === pokeTwo.id) {
-    pokeOne = getRandomPokemon(pokemonData);
-}
 
-while (pokeTwo.id === pokeThree.id) {
-    pokeTwo = getRandomPokemon(pokemonData);
-}
+function refreshPoke() {
 
-while (pokeThree.id === pokeOne.id) {
-    pokeThree = getRandomPokemon(pokemonData);
-}
+    let pokeOne = getRandomPokemon(pokemonData);
+    let pokeTwo = getRandomPokemon(pokemonData);
+    let pokeThree = getRandomPokemon(pokemonData);
+
+    while (pokeOne.id === pokeTwo.id) {
```

Comment:
You can put your while logic all in one condition.
Ex: (pokeOne.id === pokeTwo.id || (pokeTwo.id === pokeThree.id) || (pokeThree.id === pokeOne.id)){
        pokeTwo = getRandomPokemon(pokemonData);
       pokeThree = getRandomPokemon(pokemonData);
}